src/tests/actions/ArticlesActions.test.jsx

```diff
@@ -0,0 +1,41 @@
+import moxios from 'moxios';
```

Comment:
The file name shouldn't be in .jsx because the test's file is not a component but a pure javascript file